docker/hooks/post_checkout

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+export DOCKER_HUB=TRUE
```

Comment:
Should `TRUE` be uppercase or lowercase?

On the csh or zsh command line, `man true` provides a man page but `man TRUE` does not.

Reply:
doesn't matter, I guess. `ifneq` is not a shell "command". I tried with even exporting with `TRue` and `ifneq ($(DOCKER_HUB),TRue)` - works good.

Reply:
I think we will be OK but try doing the same with `false`...
```bash
bash
$ if true == "true" ; then echo yes; fi
yes
$ if false == "false" ; then echo yes; fi
<nothing>
```

Reply:
tried with `fAlse`. works like string matching for that as well.

Reply:
In bash?!?

Reply:
Not in bash, but it's easier to write an if using `ifeq()` than figuring out its bash equivalent for the Makefile.